README.md

```diff
@@ -14,7 +14,7 @@ The tool is built using open source software: [Google Earth Engine(GEE)](https:/
 
 - [Pre-processing script](https://github.com/mhscience/landslides_detection/blob/master/pre_processingGEE/pre_processing_thesis_mh.js) developed for Google Earth Engine. The script obtains cloud-free images from optical satellite imagery (Sentinel-2), extracts spectral and topographic features from Sentinel-2 and global Digital Elevation Model (DEM), and computes new landslides diagnostic features at pixel level
 
-- [Image segmentation program](https://github.com/mhscience/landslides_detection/tree/master/segmentation) developed in Python.  Image segmentation is the first step towards the application of OBIA. It consists on the subdivision of an image into spatially continuous, disjoint, and relative homogeneous regions that refer to segments. This stage is implemented as a two-step approach: (a) an initial segmentation using a [k-means script](https://github.com/mhscience/landslides_detection/tree/master/segmentation/k_means_segmentation)   (developed using [RSGISLib](https://www.rsgislib.org/)); (b) [merging algorithm script](https://github.com/mhscience/landslides_detection/tree/master/segmentation/merging_algorithm) using a region-growing implementation
```

Comment:
We cannot remove this because it is linked to the master thesis. Let me think about how can I update this properly for the current paper.

Reply:
If you like my previous suggestion, we can go ahead like that. Hear from you

Reply:
Yes, fine!